brd-android/trade/src/main/java/com/fabriik/trade/ui/features/swap/AmountConverter.kt

```diff
@@ -40,10 +41,13 @@ class AmountConverter(
         val convertedAmount = sourceAmount.multiply(rate)
 
         val destFee = estimateFee(convertedAmount, destinationCurrency, fiatCurrency)
+
         val convertedDestAmount =
-            if (destFee?.included == true) convertedAmount - destFee.cryptoAmount.divide(receivingFeeRate) else convertedAmount
+            if (destFee?.included == false) {
```

Comment:
could we review together the methods _convertDestinationCryptoToSourceCrypto_, _convertSourceCryptoToDestinationCrypto_ I think there are other cases where it will not be calculated properly?

Reply:
sure, but I am not familiar with the android code, what exactly is "included" here? Destination fee should always be deducted, even when it's not erc20 tokens